backend/spherre/tests/test_service/test_transaction_service.py

```
@@ -0,0 +1,352 @@
+from datetime import datetime
+from unittest import TestCase
+
+from spherre.app import create_app
+from spherre.app.extensions import db
+from spherre.app.models.account import Account, Member
+from spherre.app.models.transaction import Transaction, TransactionStatus, TransactionType
```

Ruff (0.11.9)

`spherre.app.models.transaction.Transaction` imported but unused

Remove unused import: `spherre.app.models.transaction.Transaction`

(F401)